### DISCUSSION TITLE
ObjectID is not 12 bytes

### DISCUSSION THREAD

**Discussion by builditpossible-gs** [prisma/prisma#21086] [Q&A]
### Bug description

I am using sveltekit, prisma, lucia, mongodb in my project. While creating new user from lucia, I get error.

`PrismaClientKnownRequestError: 
Invalid prisma.user.create() invocation: 
Malformed ObjectID: provided hex string representation must be exactly 12 bytes, instead got: "Dw1LEILpEUHbZPi", length 15 for the field 'id'.`

Can't find a way to make that id 12 Bytes. Please help me solve this issue.

### Prisma information

```prisma
// Add your schema.prisma
```

```ts
// Add your code using Prisma Client
```

### Environment & setup

- OS: 
- Database: 
- Node.js version: 

### Prisma Version

```

```

**Comment by ludralph** [comment]
Hi @sumbganesh 👋 

Thank you for raising this question.

How are you generating the value for ObjectID?

Can you also share your Prisma schema so I can look at how you defined the field that generates the ObjectID?

**Comment by alhamuddin** [comment]
```typescript
model Account {
  id                String  @id @default(auto()) @map("_id") @db.ObjectId
  userId            String  @db.ObjectId
  type              String
  provider          String
  providerAccountId String
  refresh_token     String? @db.String
  access_token      String? @db.String
  expires_at        Int?
  token_type        String?
  scope             String?
  id_token          String? @db.String
  session_state     String?

  user User @relation(fields: [userId], references: [id], onDelete: Cascade)

  @@unique([provider, providerAccountId])
  @@map("accounts")
}

enum UserRole {
  ADMIN
  USER
}

model User {
  id            String    @id @default(auto()) @map("_id") @db.ObjectId
  email         String?   @unique
  emailVerified DateTime?
  image         String?
  password      String?
  isAdmin       Boolean?
  role          UserRole  @default(USER)
  name          String
  createdAt     DateTime  @default(now())
  updatedAt     DateTime  @updatedAt
  Cart          Cart[]
  Session       Session[]
  Account       Account[]

  @@map("users")
}

```

**Reply by alhamuddin** [reply]
Can you tell me why i am getting same error mentioned in the post subject?

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
